### PR TITLE
Overcharging SMESseseses

### PR DESCRIPTION
**Pull request by TamberP**
And it's **absolutely untested** because I am a linux grognard, and the tgui here is too new, so I can't see the damn UI.

## About the PR 
Adds the ability to disable the 'safety limits' on SMES's, so you can charge them much faster than you should.
Also with the ability to render yourself, and the electrical equipment room, a shower of shrapnel and a smoking hole in the station because if they were meant to charge that fast, NT woulda set 'em to charge that fast!

## Why's this needed? 
Sometimes, you want to ram power into an SMES a bit faster! Maybe you get away with it, maybe you don't. You rolls the dice, you takes your chances. The harder you run it, the worse your chances.

## Changelog

```changelog
(u)Tamber
(*)Overcharging SMESes!
(+)Fun for all the engineering family! Except the guy stood next to it when the thing goes bang.
```

### CODE REVIEW

**Comment by TamberP** [comment]
Okay so!
Working: Opening the smes. Closing the smes. Moving the wire. Moving the wire back.

Untested because ui fuckery: whether or not you can set it to charge at eleven billionty giggawatts and blow yoursel' tae fuck.

**Comment by coolstation** [comment]
oh i totally forgot about this, was it tested?

**Comment by TamberP** [comment]
Needs the UI side of it testing, really, and then I think it's complete.
Basically:

- pre-hack, does the SMES behave as standard
- post-hack, does it behave otherwise normally unless you set a way-high input/output level
- does it explode if you charge it at, like, 1MW ('tis a random chance, but 1MW should run into it pretty quick)
- post-hack-repair, does it behave like a standard SMES again

Basically, all the stuff that needs the ability to see the control panel which won't load for me because resource fuckery. :)
